### ISSUE TITLE
Support Oreo autofill

### ISSUE THREAD

**Issue by LaurenWags** [brave/browser-android-tabs#619]
Requested via community and GitHub:
Users would like Brave to support Oreo autofill.
https://community.brave.com/t/support-for-andriod-oreo-autofill/14641 
https://github.com/brave/browser-android-tabs/issues/560

**Comment by tobia** [comment]
This is needed for 1password integration in Android.

Please test it with 1password for Android, to make sure Brave is providing the right context.

**Comment by clarktom** [comment]
This feature is lacking. Since I'm using 1password, I can't use Brave on Android

**Comment by tobia** [comment]
@clarktom Don't be so drastic. Your can use the 1Password special keyboard, or good old copy and paste. If you have many fields to fill in, you can put 1Password in split screen so that it doesn't lock up after you copy every field.

But yes, autofill would be useful.

**Comment by MKahmen** [comment]
Seems to be fixed / working in the latest android version! 👍

**Comment by clarktom** [comment]
Yes, I have also noticed it ! That's a good news 👍

**Comment by johnbillion** [comment]
With the latest Brave and the recently released 1Password 7 on Android, this appears to be working a treat!

**Comment by screendriver** [comment]
For me unfortunately not. I'm using [Enpass](https://www.enpass.io) and it works in Firefox Focus and the DuckDuckGo Browser but not in Brave :cry:

**Comment by nooz** [comment]
Autofill with Keepass2Android still not working as of v1.0.57

**Comment by rob151515** [comment]
There seems to be an issue with the implementation of autofill.

I've been doing some testing with the Brave browser on a Pixel 2 using Android Pie and autofill seems to work the first time the browser is opened from a cold start (ie. swipe it away from recents, then open it again) - you get the autofill prompt on the page.

If I then hit the home button to minimise the browser, then go and open it straight back up, the autofill prompt no longer shows. I have to then kill the browser by swiping it away in recents then open it again to get it to work.

I have also tested DuckDuckGo & Chrome browsers on the same device and they are working fine so the problem seems to be limited to this browser.

**Comment by Michi0301** [comment]
I use Engpass to store passwords. 
I can confirm that the auto fill option shows up in the Duckduckgo browser but not in Brave, so I would assume there is something missing in Brave.

**Comment by beevvy** [comment]
I can confirm that autofill doesn't work in Brave 1.0.61 with LastPass on a Pixel 2. It works just fine in Chrome 70.

**Comment by leobel96** [comment]
It works in brave 1.0.61 with bitwarden as password manager and android 9.0

**Comment by Brave-Matt** [comment]
+1 from Community:
https://community.brave.com/t/bitwarden-autofill-not-working-in-brave/39758/2

**Comment by PlanetWilson** [comment]
Autofill not working for me either on 1.0.72 and enpass on Android. Pity as it makes it pretty unusable for me.

**Comment by PlanetWilson** [comment]
So after more testing with Enpass I can get the system autofill notification to appear but not the one that should appear dropped down from the username field itself. If I do happen to go with the system autofill notification then Enpass asks what credentials I wish to pass to "Brave" rather than the URL I am visiting. i.e. it is like Brave is not passing the URL to the autofill service itself. (similar issue mentioned here https://discussion.enpass.io/index.php?/topic/5563-wrong-autofill-information-gets-added-automatically/&tab=comments#comment-11998 with a different browser)

**Comment by PlanetWilson** [comment]
Enpass and 1Password work with Chrome and Firefox Focus for me but not with Brave.

**Comment by ryscar13** [comment]
Autofill still not working for me on Brave 1.0.77 and android 8.0.

Hope a fix is around the corner as it's my only (small) gripe with Brave! Thanks!

**Comment by Rouzax** [comment]
Also not working for me on:
Brave 1.0.89
Android 9.0 Galaxy A8
1Password 7.1.3

**Comment by sjdrc** [comment]
Same issue on:
Pixel 2 XL PQ2A.190405.003
Brave 1.0.89
Bitwarden 1.22.0

**Comment by sjdrc** [comment]
Interestingly, I am able to use BitWarden autofill from Brave webview tabs (at least in some circumstances). When I try to log into a slack account with google SSO enabled through the slack app, it opens a brave custom tab (has the brave logo in top right) and offers me the regular system autofill popup. Same thing does not get offered when trying to log into gmail through the Brave app.

**Comment by Rouzax** [comment]
Still autofill saving is not working:
Brave 1.0.92
Android 9.0 Galaxy A8
1Password 7.1.4

**Comment by bdcarr** [comment]
I don't see the autofill menu appear at all in Brave.

Brave 1.0.92
Keepass2Android 1.07-r1
Android 9.0
Galaxy A50 (SM-A505F/DS)

**Comment by sys9kdr** [comment]
I have same issue. Pop-up doesn't show.

* Brave 1.0.94
* Lastpass
* Android 9.0
* Huawei P10

**Comment by Rouzax** [comment]
Is there any update/confirmation on this?

**Comment by ragregory42** [comment]
I have same issue. Autofill non-functional. 
Brave 1.0.99
Bitwarden Autofill Framework/Bitwarden v2.0.6
Android 8.0.0
LG V30

**Comment by josephsteele** [comment]
Same issue for me:
 Brave v1.0.99
Bitwarden v2.0.6
Android 9
Samsung S8

**Comment by dunxd** [comment]
I'm finding the autofill works very intermittently with the following:
Brave v1.1.1
Bitwarden 2.1.0
Android 9
Motorola One

I frequently have to open Bitwarden, search, copy password and paste into Brave. Occaisionaly I get the pop-up under a login field - but more often than not I don't get it.

**Comment by hoejmann** [comment]
Have never been able to get the autofill option to appear.
Google Pixel 2
1 Password 7.3
Brave 1.2
Android 9

**Comment by ihatemicrosoft-was-already-taken** [comment]
Please see also:

https://community.brave.com/t/closed-enpass-choose-wrong-link-url/666
https://community.brave.com/t/android-enpass-autofill-not-working-wrong-url-handover/15542
https://community.brave.com/t/android-enpass-autofill-not-working-properly-due-to-wrong-url-handover/72243

thanks

**Comment by RobinJ1995** [comment]
Android 9 on OnePlus 5T. Same issue. This is a blocker for me. Supporting password managers is a pretty basic requirements for a browser

**Comment by Kallb123** [comment]
Android 9 with OnePlus 3. Agree with above. Password manager autofill is almost a requirement to browse the Web safely.

**Comment by sys9kdr** [comment]
AFAIK, Oreo autofill is not supported by Chrome, Opera and other chrome-based browsers on Android（Oreo~Pie）.
FIrefox Preview supports Orea autofill.

I use legacy autofill with LastPass instead of Oreo autofill. It's not good but enough.

**Comment by Rouzax** [comment]
It seems to be getting worse and more and more sites will not work anymore with the autofill on Brave with 1Password. Also the legacy autofill is not showing up all the time.

**Comment by dstronczak** [comment]
Is there any time horizon when this could be fixed? This is a huge blocker.

**Comment by Rouzax** [comment]
> Is there any time horizon when this could be fixed? This is a huge blocker.
Agreed, this together with the really spotty sync is starting to make me doubt my switch to Brave.
